### DISCUSSION TITLE
游戏又崩了

### DISCUSSION THREAD

**Discussion by acwuy** [Meloong-Git/PCL#1208] [提问] [answered]
我今天特别好奇，于是打开了PCL的一些文件看，然后关闭后又打开喷气背包猫整合包发现出现了一个很长的错误报告，我的游戏无法再次打开了，每个整合包都不可以，原版也不可以，我很困惑，所以我删掉了PCL重新下载了一遍，然后就好了，这应该是我的问题吧

**Comment by Creependerman** [comment]
可以发一下错误报告吗，谢谢

**Comment by WorldHim** [comment, marked as answer]
> 我今天特别好奇，于是打开了PCL的一些文件看，然后关闭后又打开喷气背包猫整合包发现出现了一个很长的错误报告，我的游戏无法再次打开了，每个整合包都不可以，原版也不可以，我很困惑，所以我删掉了PCL重新下载了一遍，然后就好了，这应该是我的问题吧

如果你通过外部工具对pcl的文件进行更改有可能导致配置错误甚至使游戏无法启动。如果你将pcl的文件重置后仍然无法恢复正常建议提交issue。
